tests/wallet/rpc/test_dl_wallet_rpc.py

```diff
@@ -0,0 +1,161 @@
+import asyncio
+import logging
+import pytest
+
+from chia.consensus.block_rewards import calculate_base_farmer_reward, calculate_pool_reward
+from chia.data_layer.data_layer_wallet import SingletonRecord
+from chia.rpc.full_node_rpc_api import FullNodeRpcApi
+from chia.rpc.rpc_server import start_rpc_server
+from chia.rpc.wallet_rpc_api import WalletRpcApi
+from chia.rpc.wallet_rpc_client import WalletRpcClient
+from chia.simulator.simulator_protocol import FarmNewBlockProtocol
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.types.peer_info import PeerInfo
+from chia.util.ints import uint16, uint32, uint64
+from tests.setup_nodes import bt, setup_simulators_and_wallets, self_hostname
+from tests.time_out_assert import time_out_assert
+from tests.util.rpc import validate_get_routes
+
+log = logging.getLogger(__name__)
+
+
+class TestWalletRpc:
```

Comment:
Tests don't need classes.  They just use up indentation and add unused `self` parameters.

Reply:
Yeah sorry this is a bad habit of mine, and propagates itself because I'll often just copy the test boilerplate from other tests.